comps/dataprep/neo4j/llama_index/extract_graph_neo4j.py

```diff
@@ -465,18 +552,19 @@ def ingest_data_to_neo4j(doc_path: DocPath):
             logger.info(f"An error occurred while verifying the API Key: {e}")
     else:
         logger.info("NO OpenAI API Key. TGI/TEI endpoints will be used.")
-        llm_name = get_model_name_from_tgi_endpoint(TGI_LLM_ENDPOINT)
+        llm_name = get_attribute_from_tgi_endpoint(TGI_LLM_ENDPOINT, "model_id")
         llm = TextGenerationInference(
             model_url=TGI_LLM_ENDPOINT,
             model_name=llm_name,
             temperature=0.7,
             max_tokens=1512,  # 512otherwise too shor
```

Comment:
`512otherwise too shor` -> `512 otherwise too short`
Also is it `512` or `1512`?
Is `512` like min value?